src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -496,7 +496,13 @@ public DebuggedProcess(bool bLaunched, LaunchOptions launchOptions, ISampleEngin
                     }
                 }
 
-
+                var arch = await MICommandFactory.GetTargetArchitecture();
+                if (arch == TargetArchitecture.Unknown)
```

Comment:
the trouble is that since we never checked the architecture value against the target type it was often specified incorrectly. For example, in the first VSIX for Linux debugging it is hardcoded to arm. So I think 1 is not a good choice. We could do 2, but I would be opposed to halting the debugging session for this reason. Were you thinking of a popup for this, or just an error message to the output window? @gregg-miskelly